safe/definitions/concepts.py

```diff
@@ -469,6 +469,33 @@
             }
         ],
     },
+    'thresholds': {
+        'key': 'thresholds',
+        'description': tr(
+            'A range that defined with minimum and maximum value. In InaSAFE '
+            'we exclude the minimum value but include the maximum value. In '
+            'mathematical expression: minimum value < x <= maximum value. It '
+            'is used for doing classification for continuous data.'),
+        'citations': [
+            {
+                'text': '',
+                'link': u''
+            }
+        ],
+    },
+    'value_maps': {
+        'key': 'value_maps',
+        'description': tr(
+            'A mapping of a set of values to another set of unique values. '
+            'Each unique value represents a particular class. It is used for '
+            'doing classification for classified (non-continuous) data'),
```

Comment:
'A conceptual mapping between one set of unique values and another set of unique values. Each unique value represents a particular class. It is used to express terms or concepts from one classification system in another classification system and only applies to non-continuous data. For example a value map can be used to express local 
names for entities (e.g. street type: "jalan") into generic concepts (e.g. street type: "residential").'